### DISCUSSION TITLE
Virus Total

### DISCUSSION THREAD

**Discussion by Xfire08** [ente-io/ente#1358] [Q&A] [answered]
Just wanted to understand why is the virus total flagging ente auth windows exe. is it safe to use ?

![Virus Total Scan](https://github.com/ente-io/ente/assets/14965824/1fd8fce1-15d5-41e5-93a5-d69fbb37aa57)

**Comment by vishnukvmd** [comment]
@prateekmedia is there anything we've to do here?

**Comment by prateekmedia** [comment, marked as answer]
It is completely safe to use, the app is already verified using Windows Defender security.
Submission id: b381dddf-07ef-4ed1-bb11-74eaebedc249

For reference the two that flagged are already giving false postives:
For Bkav pro: https://www.reddit.com/r/antivirus/comments/14zijs6/hi_is_this_a_virus/
For Trapmine: https://www.reddit.com/r/antivirus/comments/15zfn7x/maliciousmoderatemlscore/

So they are only false positives and seems like only calculated using some ml algorithm and not actual detection.

See https://github.com/ente-io/ente/issues/658
